plugins/catalog-import/src/components/StepPrepareCreatePullRequest/PreviewCatalogInfoComponent.test.tsx

```diff
@@ -104,4 +117,30 @@ describe('<PreviewCatalogInfoComponent />', () => {
     expect(kindText).toBeInTheDocument();
     expect(kindText).not.toBeVisible();
   });
+
+  it('renders with custom catalog filename', () => {
+    render(
+      <ApiProvider
+        apis={TestApiRegistry.from([
```

Comment:
Using the `<TestApiProvider>` here lets you use fewer characters :grin:

Reply:
Nice! Converted all tests in this file to use it